Demo/src/main/java/com/braintreepayments/demo/PayPalFragment.java

```diff
@@ -65,24 +108,77 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         return view;
     }
 
+    @OptIn(markerClass = ExperimentalBetaApi.class)
     @Override
     public void onResume() {
         super.onResume();
         PayPalPendingRequest.Started pendingRequest = getPendingRequest();
         if (pendingRequest != null) {
-            PayPalPaymentAuthResult paymentAuthResult = payPalLauncher.handleReturnToApp(pendingRequest, requireActivity().getIntent());
-            if (paymentAuthResult instanceof PayPalPaymentAuthResult.Success) {
-                completePayPalFlow((PayPalPaymentAuthResult.Success) paymentAuthResult);
+            PayPalPaymentAuthResult paymentAuthResult = payPalLauncher.handleReturnToApp(
+                    pendingRequest,
+                    requireActivity().getIntent()
+            );
+
+            if (paymentAuthResult != null) {
```

Comment:
Can this be null? I don't think the `handleReturnToApp` returns an optional so it should always exist